### PR TITLE
Main

### PR DESCRIPTION
**Pull request by annegeorge17**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Comment by annegeorge17** [comment]
To whoever reviews this PR, the Swagger ModelValidation errors need more clarification. From my view I don't see an additional property as both the spec and the example contain just the disabled property. Could I get an official answer regarding this?

**Comment by raosuhas** [comment]
PLease use the template above to create the PR for control plane PRs

**Comment by raosuhas** [comment]
Please also follow the instructions in the control PLane PR description and add a previous version as a base commit so that reviewers can diff the changes across versions

File: .github/PULL_REQUEST_TEMPLATE/control_plane_template.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Please understand this diagram before proceeding. It explains how to get your PR
 What's the purpose of this PR? Check the specific option that applies. This is **mandatory**!
 
   - [ ] New resource provider.
-  - [ ] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been created in adherence to [OpenAPI specs PR creation guidance](https://aka.ms/azsdkdocs/createopenapispec)).
+  - [X] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been created in adherence to [OpenAPI specs PR creation guidance](https://aka.ms/azsdkdocs/createopenapispec)).
```
Comment (weshaggard) [inline .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:17]: You shouldn't be editing this template as part of your change.

Comment (annegeorge17) [reply .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:17]: Oh oops, do I create a new document with the .md? I was unsure how to use the template per the above commenter

Comment (weshaggard) [reply .github/PULL_REQUEST_TEMPLATE/control_plane_template.md]: These should go in your PR description. It looks like you didn't select the template at PR creation time. You will want to create a new PR and select the correct template as the description says.

Comment (annegeorge17) [reply .github/PULL_REQUEST_TEMPLATE/control_plane_template.md]: @weshaggard Hi! I've tried making a new PR but each time at creation it doesn't use the template. If I can leave a comment with the same information is that sufficient? If not, could you maybe write down the steps to see if I'm doing something wrong?

Comment (weshaggard) [reply .github/PULL_REQUEST_TEMPLATE/control_plane_template.md]: Before you hit the create button did you click the preview tab in the description and then click the control plane template link? It should reload your PR with the correct template and then you click the create button.

**Pull request closed without merging**